src/openforms/authentication/contrib/yivi_oidc/tests/test_auth_flow_init.py

```diff
@@ -0,0 +1,234 @@
+        signicat_scope = match.group(1) if match else None
+
+        assert signicat_scope is not None
+
+        base64_part = signicat_scope.split(":")[-1]
+
+        # Decode to a condiscon array
+        decoded_bytes = base64.b64decode(base64_part)
+        decoded_str = decoded_bytes.decode("utf-8")
+        condiscon = json.loads(decoded_str)
+
+        self.assertEqual(condiscon, expected)
+
+    @mock_yivi_config(pseudo_claim=["attribute.pseudo"])
+    def test_start_flow_redirects_to_oidc_provider(self):
+        form = FormFactory.create(
+            authentication_backend="yivi_oidc",
+            authentication_backend__options={
+                "authentication_options": [],
+                "additional_attributes_groups": [],
+            },
+        )
+        start_url = URLsHelper(form=form).get_auth_start(plugin_id="yivi_oidc")
+
+        response = self.app.get(start_url)
+
+        self.assertEqual(response.status_code, 302)
+        redirect_target = furl(response["Location"])
+        query_params = redirect_target.query.params
+
+        self.assertEqual(redirect_target.host, "localhost")
+        self.assertEqual(redirect_target.port, 8080)
+        self.assertEqual(
+            redirect_target.path,
+            "/realms/test/protocol/openid-connect/auth",
+        )
+        self.assertEqual(
+            query_params["scope"],
+            ("openid signicat:param:condiscon_base64:W1tbImF0dHJpYnV0ZS5wc2V1ZG8iXV1d"),
+        )
+        self.assertEqual(query_params["client_id"], "testid")
+        self.assertEqual(query_params["redirect_uri"], self.CALLBACK_URL)
+
+    @mock_yivi_config(bsn_claim=["attribute.bsn"])
+    def test_signicat_condiscon_contains_only_the_chosen_authentication_and_additional_attributes(
+        self,
+    ):
+        AttributeGroupFactory(name="personal", attributes=["first_name", "last_name"])
+        AttributeGroupFactory(name="mail", attributes=["email_address"])
+        AttributeGroupFactory(name="phone", attributes=["phone_number"])
+        form = FormFactory.create(
+            authentication_backend="yivi_oidc",
+            authentication_backend__options={
+                "authentication_options": [AuthAttribute.bsn],
+                "additional_attributes_groups": ["personal", "mail"],
+            },
+        )
+        start_url = URLsHelper(form=form).get_auth_start(plugin_id="yivi_oidc")
+        response = self.app.get(start_url)
+
+        self.assertEqual(response.status_code, 302)
+        redirect_target = furl(response["Location"])
+        query_params = redirect_target.query.params
+
+        self.assertEqual(
+            query_params["scope"],
+            (
+                "openid "
+                "signicat:param:condiscon_base64:"
+                "W1tbImF0dHJpYnV0ZS5ic24iXV0sIFtbImZpcnN0X25hbWUiLCAibGFzdF9u"
+                "YW1lIl0sIFtdXSwgW1siZW1haWxfYWRkcmVzcyJdLCBbXV1d"
+            ),
+        )
```

Comment:
probably you rather want to assert that `openid` is in `scope.split(" ")` - the order is not relevant afaik and you have the condiscon assert below anyway. I don't find the asserts on the raw base64 strings very meaningful.